src/Apps/Artwork/ArtworkApp.tsx

```diff
@@ -217,7 +226,15 @@ export const ArtworkApp: React.FC<Props> = props => {
       <ArtworkMetaFragmentContainer artwork={artwork} />
 
       <ArtworkTopContextBarFragmentContainer artwork={artwork} />
-
+      <GridColumns>
```

Comment:
I think the `displayAuctionCreateAlertHeader` boolean should come outside and wrap this whole `<GridColumns />` component, otherwise in cases where it shouldn't be enabled, the layout has this extra empty div rendered.

Reply:
I see the feature flag (which also controls the display of this component) is being checked inside that component, which can also result in an empty `<GridColumns />` component here in the layout.

I don't think it's a huge deal, but I _think_ we should try to not render empty div's from the layout if possible.

Can the feature flag check be moved here as well? (And all the checks can be wrapped up in a boolean and checked _before_ the `<GridColumns />` is rendered).

Reply:
Yep, agree with the empty div. The feature flag was initially there (as usual) but it was unreachable for testing since the tests were within my component, here is [some context](https://github.com/artsy/force/pull/12678#discussion_r1280869336) on this decision since I discussed with George as well. We thought it would be better to keep the tests within the component than on an ArtworkApp level and also to have everything in one place which can easily be removed.

Reply:
Ah, catching up. Reading that....and I agree with @dariakoko and disagree with @gkartalis 😆  I think it's better for the component to _not_ know about the feature flag here, and for the parent/layout to know about it since it can make sure there are no empty div's or layout weirdness in that case. I don't think it's a big deal to clean it up when it's time to retire the feature flag, it's just a line of code here or in the component - prob shouldn't influence our implementation.

Re: testing - I also don't think it's a big deal - straightforward test for the banner component, and then a test in `ArtworkApp` to assert the banner component will be included (or not) in the DOM. We'd still want a test in `ArtworkApp` that checks the component is being rendered for a closed auction work, and accessing the feature flag mock in the `ArtworkApp` file seems like something we should be able to do as well.